contrib/uwsgi.ini

```diff
@@ -38,3 +38,7 @@ log-5xx = true
 
 ; Number of uWSGI workers to spawn. This should typically be 2n+1, where n is the number of CPU cores present.
 ; processes = 5
+
+; If using subdirectory hosting e.g. example.com/nautobot, you must uncomment this line. Otherwise you'll get double paths e.g. example.com/nautobot/nautobot/.
+; See: https://uwsgi-docs.readthedocs.io/en/latest/Changelog-2.0.11.html#fixpathinfo-routing-action
+; route-run = fixpathinfo:
```

Comment:
Actually, can we use this PR to get rid of the `contrib` files? With the move to `pip` installation, they aren't readily accessible by the user anyway (I think they're now somewhere like `/opt/nautobot/lib/python3.8/site-packages/nautobot/contrib/`??) and it's just tech debt to have to keep these in sync with the examples provided in the docs.

Reply:
Yes actually. I'll go ahead and do that.